packages/css/webpack.config.js

```diff
@@ -0,0 +1,43 @@
+const baseConfig = require('@ovh-ux/ui-kit.config/src/webpack.dist.config');
+const merge = require('webpack-merge');
+const path = require('path');
+
+const ExtraneousFileCleanupPlugin = require('webpack-extraneous-file-cleanup-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+module.exports = merge(baseConfig, {
+  entry: {
+    'oui-core': [
+      './src/oui-core.less',
+    ],
+    'oui-flags': [
+      './src/oui-flags.less',
+    ],
+    'oui-icons': [
+      './src/oui-icons.less',
+    ],
+    oui: [
+      './src/index.less',
+    ],
+  },
+  output: {
+    filename: '[name].js',
```

Comment:
**question**:
Do you need to produce js file output for this package ? I mean, can we have just a css output file here, as that is what the package is supposed to provide ?